pyomo/contrib/mindtpy/util.py

```diff
@@ -106,6 +101,18 @@ def add_feas_slacks(m, config):
                 MindtPy.feas_opt.feas_constraints.add(
                     constr.body - constr.lower >= -MindtPy.feas_opt.slack_var
                 )
+    # Setup objective function for the feasibility subproblem.
+    if config.feasibility_norm == 'L1':
+        MindtPy.feas_obj = Objective(
+            expr=sum(s for s in MindtPy.feas_opt.slack_var[...]), sense=minimize
```

Comment:
This syntax works, but you could also do `Mindtpy.feas_opt.slack_var.values()`, which might be a bit more clear

Reply:
Great. I will change this to `slack_var.values()`.

Reply:
I think there might be another couple places you could make this change too. Maybe do a search for `...`